django_mongodb/fields/auto.py

```diff
@@ -2,15 +2,11 @@
 from django.core import exceptions
 from django.db.models.fields import AutoField
 from django.utils.functional import cached_property
-from django.utils.translation import gettext_lazy as _
 
+from .objectid_mixin import ObjectIdMixin
 
-class ObjectIdAutoField(AutoField):
-    default_error_messages = {
-        "invalid": _("“%(value)s” value must be an Object Id."),
-    }
-    description = _("Object Id")
 
+class ObjectIdAutoField(AutoField, ObjectIdMixin):
```

Comment:
swap order of AutoField and ObjectIdMixin. Method resolution order is left to right.